verticapy/utilities.py

```diff
@@ -1831,12 +1896,16 @@ def read_csv(
             input_relation = "v_temp_schema.{}".format(quote_ident(table_name))
         file_header = []
         path_first_file_in_folder = path
-        if multiple_files:
+        if multiple_files and ingest_local:
             path_first_file_in_folder = get_first_file(path, "csv")
-        if not (header_names) and not (dtype) and (compression == "UNCOMPRESSED"):
+        if not (header_names) and not (dtype) and (compression == "UNCOMPRESSED") and ingest_local:
             if not (path_first_file_in_folder):
                 raise ParameterError("No CSV file detected in the folder.")
             file_header = get_header_name_csv(path_first_file_in_folder, sep)
+        elif not (header_names) and not (dtype) and (compression != "UNCOMPRESSED"):
+            raise ParameterError("The input file is compressed and parameters 'dtypes' and 'header_names' are not defined. It is impossible to read the file's header.")
+        elif not (header_names) and not (dtype) and not(ingest_local):
+            raise ParameterError("The input file is in the Vertica server and parameters 'dtypes' and 'header_names' are not defined. It is impossible to read the file's header.")
```

Comment:
a) "dtype" instead of "dtypes"
b) same as previous one...
The header of the input file cannot be read as the file is located on the vertica server. Please specify dtypes or header_names.

Reply:
Done;